test/CodeGeneration/CUDACtorDtor.C

```diff
@@ -0,0 +1,73 @@
+//------------------------------------------------------------------------------
+// CLING - the C++ LLVM-based InterpreterG :)
+//
+// This file is dual-licensed: you can choose to license it under the University
+// of Illinois Open Source License or the GNU Lesser General Public License. See
+// LICENSE.TXT for details.
+//------------------------------------------------------------------------------
+
+// The Test checks, if the symbols __cuda_module_ctor and __cuda_module_dtor are
+// unique for every module. Attention, for a working test case, a cuda
```

Comment:
How are we planning to provide that fatbin file and what is it's corresponding CUDA code?